ipaserver/install/server/install.py

```diff
@@ -1150,6 +1150,11 @@ def uninstall(installer):
                      'but it may\n'
                      'mean your system hasn\'t be restored to its '
                      'pre-installation state.', SYSRESTORE_DIR_PATH)
+    else:
+        # cleanup remaining sysrestore artifacts but keep the directory
+        for basedir, dirnames, filenames in os.walk(SYSRESTORE_DIR_PATH):
+            for filename in filenames:
+                os.unlink(os.path.join(basedir, filename))
```

Comment:
I'm unsure about this bit. A few lines up a line is printed that files remain and then they are removed. Ideally all files are removed/restored during uninstallation. If there are any left over then that is the bug and cleaning them up just covers it up.